data-raw/process_data_sets.R

```diff
@@ -272,3 +272,22 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+# Electric vehicle charging sessions
+# information from 3,395 high resolution electric vehicle charging sessions
+# Original data from: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/NFPQLW
+ev_charging <- 
+  "https://dataverse.harvard.edu/api/access/datafile/3998649?format=original&gbrecs=true" %>%
+  read_csv() %>%
+  mutate(facilityType = case_when(
```

Comment:
The variables in this dataset currently use camel case in their naming. For consistency, please use snake case. `janitor::clean_names()` should do this automatically.